web/router.ex

```diff
@@ -84,10 +81,19 @@ defmodule CodeCorps.Router do
     resources "/roles", RoleController, only: [:create]
     resources "/role-skills", RoleSkillController, only: [:create, :delete]
     resources "/skills", SkillController, only: [:create]
+    resources "/stripe-customers", StripeCustomerController, only: [:show, :create]
     resources "/tasks", TaskController, only: [:create, :update]
     resources "/users", UserController, only: [:update]
     resources "/user-categories", UserCategoryController, only: [:create, :delete]
     resources "/user-roles", UserRoleController, only: [:create, :delete]
     resources "/user-skills", UserSkillController, only: [:create, :delete]
   end
+
+  scope "/", CodeCorps, host: "api." do
+    pipe_through [:api, :bearer_auth, :current_user]
+
+    get "/:slug", SluggedRouteController, :show
+    get "/:slug/projects", ProjectController, :index
+    get "/:slug/:project_slug", ProjectController, :show
+  end
```

Comment:
Why don't we reorder authenticated routes above this so we're not separating the same pipeline?